packages/ui/src/color-modes/utils.ts

```diff
@@ -0,0 +1,69 @@
+import { Theme } from '../theme';
+import {
+  ColorModesInterface,
+  ColorsStringLiteral,
+  ColorModeTypes,
+  Color,
+  ThemeColorsStringLiteral,
+} from './types';
+import { themeGet } from '@styled-system/theme-get';
+
+export const colorGet = (path: string, fallback?: string): ((props: any) => any) =>
+  themeGet('colors.' + path, fallback);
+
+const colors = (props: { theme: Theme }): ColorModesInterface => ({
+  light: {
+    [Color.Accent]: colorGet('blue')(props),
+    [Color.Bg]: 'white',
+    [Color.BgAlt]: colorGet('ink.50')(props),
+    [Color.BgLight]: 'white',
+    [Color.Invert]: colorGet('ink')(props),
+    [Color.TextHover]: colorGet('blue')(props),
+    [Color.TextTitle]: colorGet('ink')(props),
+    [Color.TextCaption]: colorGet('ink.600')(props),
+    [Color.TextBody]: colorGet('ink.900')(props),
+    [Color.InputPlaceholder]: colorGet('ink.400')(props),
+    [Color.Border]: 'rgb(229, 229, 236)',
+    [Color.FeedbackAlert]: colorGet('orange')(props),
+    [Color.FeedbackError]: colorGet('red')(props),
+    [Color.FeedbackSuccess]: colorGet('green')(props),
+  },
+  dark: {
```

Comment:
Great stuff. This migration has been in the back of my mind. Now, the forefront.